src/javascripts/helpers/helpers.js

```diff
@@ -0,0 +1,5 @@
+export const formatNumber = (number) => {
+    return number.toLocaleString("en-US", {
+        minimumFractionDigits: 2,
+    });
+  };
```

Comment:
Add empty space at the end of file

Reply:
I fixed it @phuocduong-agilityio  . Please help me check. Thanks !